vhdbuilder/packer/test/run-test.sh

```diff
@@ -99,6 +99,11 @@ else
     TARGET_COMMAND_STRING+="--security-type TrustedLaunch --enable-secure-boot true --enable-vtpm true"
   fi
 
+  if [[ "${OS_TYPE}" == "Linux" && "${IMG_SKU}" == "20_04-lts-cvm" ]]; then
+    TARGET_COMMAND_STRING="--size Standard_EC8ads_v5"
+    TARGET_COMMAND_STRING+=" --security-type ConfidentialVM --enable-secure-boot true --enable-vtpm true --os-disk-security-encryption-type VMGuestStateOnly --specialized"
```

Comment:
is there a reason we son't make this one line since we are just adding strings?

Reply:
Haha good catch thank you!